Gruntfile.js

```diff
@@ -2,39 +2,41 @@ const webpackConfig = require('./webpack.config')
 
 module.exports = function (grunt) {
   grunt.loadNpmTasks('grunt-contrib-clean')
-  grunt.loadNpmTasks('grunt-contrib-uglify')
   grunt.loadNpmTasks('grunt-webpack')
 
   grunt.initConfig({
     clean: ['dist'],
-    uglify: {
-      my_target: {
-        files: {
-          'dist/leanplum.min.js': 'dist/leanplum.js',
-          'dist/sw/sw.min.js': 'dist/sw/sw.js'
-        }
-      }
-    },
     watch: {
       js: {
         files: [
           'lib/leanplum.js'
         ],
-        tasks: ['lint', 'compile']
+        tasks: ['build']
       }
     },
     webpack: {
       options: {
-        stats: !process.env.NODE_ENV || process.env.NODE_ENV === 'development'
+        stats: (!process.env.NODE_ENV || process.env.NODE_ENV === 'development') && {
+          all: false,
+          assets: true,
+          chunks: false,
+          entrypoints: false,
+          errorDetails: false,
+          errors: true,
+          excludeAssets: [ /\.d\.ts$/ ],
+          hash: false,
+          logging: 'error',
+          performance: false,
+          warnings: false,
+        }
```

Comment:
Can we move this to `webpack.config`? This way the whole config will be in one file, instead of separated.

Reply:
Thanks, I'll address this during the work on migration to Webpack.